pkg/agent/cniserver/server.go

```diff
@@ -381,8 +381,8 @@ func (s *CNIServer) CmdAdd(ctx context.Context, request *cnipb.CniCmdRequest) (*
 		}
 	}()
 
-	s.containerAccess.lockContainer(cniConfig.ContainerId)
```

Comment:
do any of the comments for the `containerAccessArbitrator` struct and its methods need to be updated, or are they still accurate?

Reply:
It is still accurate. We still use the lock on a containerID to prevent other goroutines. The only difference is Windows is using the infra(sandbox) container ID in the lock.